components/bl-radio-buttons-list-component/src/index.js

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
+import { RadioButton } from './radio-button';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function RadioButtonsListComponent({ component, eventHandlers }) {
+  const { classList, style, display, disabled, options, checkedValue } = component;
```

Comment:
checkedValue and selectedValue looks strange

What about value and handlerValue